docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,160 @@
+
+A visualization is worth a thousand log lines, and {kib} provides
+many options for showcasing your data. Use <<visualize, Visualize>> to create
+classic line charts, bar graphs, pie charts, histograms, and tables
+(just to name a few).
+
+Once your visualizations are just the way you want,
+use <<dashboard, Dashboard>> to collect them in one place. A dashboard provides
+insights into your data from multiple perspectives, and you can watch the data
+change in real time.
+
+[role="screenshot"]
+image::images/intro-dashboard.png[]
+
+[float]
+[[more-ways-to-visualize]]
+=== Even more ways to visualize
+
+{kib} also has these features that address specific visualization needs.
+
+* <<lens, Lens>> provides a drag-and-drop experience that helps
+you get insights from your data in just a few clicks. If there is a
+better visualization for your data, Lens suggests it, allowing
+for quick switching between visualization types.
+
+* <<canvas, Canvas>> gives you the ability to present your data in a
+visually compelling, pixel-perfect report. Give your data the “wow” factor
+needed to impress your CEO or to captivate people with a big-screen display..
+
+* <<maps, Elastic Maps>> enables you to ask (and answer) meaningful
+questions of your location-based data. Elastic Maps supports multiple
+layers and data sources, mapping of individual geo points and shapes,
+and dynamic client-side styling.
+
+* <<TSVB, TSVB>> allows you to combine
+an infinite number of aggregations to display complex data in a meaningful way.
+With TSVB, you can analyze multiple index patterns and customize
+every aspect of your visualization. Choose your own date format and color
+gradients, and easily switch your data view between time series, metric,
+top N, gauge, and markdown.
+
+[float]
+[[organize-and-secure]]
+=== Organize & secure
+
+Want to share Kibana’s goodness with other people or teams? You can do so with
+<<xpack-spaces, Spaces>>, built for organizing your visualizations, dashboards, and indices.
+Think of a space as its own mini {kib} installation &mdash; it’s isolated from
+all other spaces, so you can tailor it to your specific needs without impacting others.
+
+You can even choose which features to enable within each space. Don’t need
+Machine learning in your “Executive” space? Simply turn it off.
+
+[role="screenshot"]
+image::images/intro-spaces.jpg[]
+
+You can take this all one step further with Kibana’s security features, and
+control which users have access to each space. {kib} allows for fine-grained
+controls here, so you can give a user read-only access to
+dashboards in one space, but full access to all of Kibana’s features in another.
+
+[float]
+[[manage-all-things-stack]]
+=== Manage all things Elastic Stack
+
+<<management, Management>> provides guided processes for managing all
+things Elastic Stack &mdash; indices, clusters, licenses, UI settings, index patterns,
+spaces, and more. Want to update your {es} indices? Set user roles and privileges?
```

Comment:
Since you already talk about spaces in `Organize & secure`, I would leave it out here.